File: MenuList.java
import java.util.Scanner;

public class MenuList {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.println("1. Phone book");
        System.out.println("2. Messages");
        System.out.println("3. Chat");
        System.out.println("4. Call register");
        System.out.println("5. Tones");
        System.out.println("6. Settings");
        System.out.println("7. Call divert");
        System.out.println("8. Music");
        System.out.println("9. Games");
        System.out.println("10. Calculator");
        System.out.println("11. Reminders");
        System.out.println("12. Clock");
        System.out.println("13. Profiles");
        System.out.println("14. WAP Services");
        System.out.println("15. SIM Services");

        System.out.print("Enter main menu: ");
        int mainMenu = input.nextInt();

        switch(mainMenu) {
            case 1 -> {
                System.out.println("Phone book:");
                System.out.println("1 Search");
                System.out.println("2 Service Nos.");
                System.out.println("3 Add name");
                System.out.println("4 Erase");
                System.out.println("5 Edit");
                System.out.println("6 Assign tone");
                System.out.println("7 Send b card");
                System.out.println("8 Options");
                System.out.println("9 Speed dials");
                System.out.println("10 Voice tags");
                System.out.print("Enter: ");
                int phonebookoptions = input.nextInt();
                switch(phonebookoptions) {
                    case 1 -> System.out.println("Search");
                    case 2 -> System.out.println("Service Nos.");
                    case 3 -> System.out.println("Add name");
                    case 4 -> System.out.println("Erase");
                    case 5 -> System.out.println("Edit");
                    case 6 -> System.out.println("Assign tone");
                    case 7 -> System.out.println("Send b card");
                    case 8 -> {
                        System.out.println("1 Type of view");
                        System.out.println("2 Memory status");
                        System.out.print("Enter: ");
                        int options = input.nextInt();
                        switch(options) {
                            case 1 -> System.out.println("Type of view");
                            case 2 -> System.out.println("Memory status");
                        }
                    }
                    case 9 -> System.out.println("Speed dials");
                    case 10 -> System.out.println("Voice tags");
                }
            }
            case 2 -> {
                System.out.println("Messages:");
                System.out.println("1 Write messages");
                System.out.println("2 Inbox");
                System.out.println("3 Outbox");
                System.out.println("4 Picture messages");
                System.out.println("5 Templates");
                System.out.println("6 Smileys");
                System.out.println("7 Message settings");
                System.out.println("8 Info service");
                System.out.println("9 Voice mailbox number");
                System.out.println("10 Service command editor");
                System.out.print("Enter: ");
                int messages = input.nextInt();
                switch(messages) {
                    case 1 -> System.out.println("Write messages");
                    case 2 -> System.out.println("Inbox");
                    case 3 -> System.out.println("Outbox");
                    case 4 -> System.out.println("Picture messages");
                    case 5 -> System.out.println("Templates");
                    case 6 -> System.out.println("Smileys");
                    case 7 -> {
                        System.out.println("1 Set 1");
                        System.out.println("2 Common");
                        int set1 = input.nextInt();
                        switch(set1) {
                            case 1 -> {
                                System.out.println("1 Message centre number");
                                System.out.println("2 Messages sent as");
                                System.out.println("3 Message validity");
                            }
                            case 2 -> {
                                System.out.println("1 Delivery reports");
                                System.out.println("2 Reply via same centre");
                                System.out.println("3 Character support");
                            }
                        }
                    }
                    case 8 -> System.out.println("Info service");
                    case 9 -> System.out.println("Voice mailbox number");
                    case 10 -> System.out.println("Service command editor");
                }
            }
            case 3 -> System.out.println("Chat");
            case 4 -> {
                System.out.println("Call register:");
                System.out.println("1 Missed calls");
                System.out.println("2 Received calls");
                System.out.println("3 Dialled numbers");
                System.out.println("4 Erase recent call lists");
                System.out.println("5 Show call duration");
                System.out.println("6 Show call costs");
                System.out.println("7 Call cost settings");
                System.out.println("8 Prepaid credit");
                System.out.print("Enter: ");
                int callregister = input.nextInt();
                switch(callregister) {
                    case 1 -> System.out.println("Missed calls");
                    case 2 -> System.out.println("Received calls");
                    case 3 -> System.out.println("Dialled numbers");
                    case 4 -> System.out.println("Erase recent call lists");
                    case 5 -> {
                        System.out.println("1 Last call duration");
                        System.out.println("2 All calls duration");
                        System.out.println("3 Received calls duration");
                        System.out.println("4 Dialled calls duration");
                        System.out.println("5 Clear timers");
                        int callduration = input.nextInt();
                        switch(callduration) {
                            case 1 -> System.out.println("Last call duration");
                            case 2 -> System.out.println("All calls duration");
                            case 3 -> System.out.println("Received calls duration");
                            case 4 -> System.out.println("Dialled calls duration");
                            case 5 -> System.out.println("Clear timers");
                               }
                        }
                    case 6 -> { 
                        System.out.println("1 Last call cost");
                        System.out.println("2 All calls’ cost");
                        System.out.println("3 Clear counters");
                        int showcallcost = input.nextInt();
                        switch(showcallcost ){
                            case 1 -> System.out.println("Last call cost");
                            case 2 -> System.out.println("All calls’ cost");
                            case 3 -> System.out.println("Clear counters");
                            }
                        }
                    case 7 -> {
                        System.out.println("1 Call cost settings");
                        System.out.println("2 Show cost limit");
                        int callcostsettings = input.nextInt();
                        switch(callcostsettings){
                            case 1 -> System.out.println("Call cost settings");
                            case 2 -> System.out.println("Show cost limit");
                        }
                    }
                 case 8 -> System.out.println("Prepaid credit");
                }
            }
            case 5 -> {
                System.out.println("Tones:");
                System.out.println("1 Ringing tone");
                System.out.println("2 Ringing volume");
                System.out.println("3 Incoming call alert");
                System.out.println("4 Composer");
                System.out.println("5 Message alert tone");
                System.out.println("6 Keypad tones");
                System.out.println("7 Warning and game tones");
                System.out.println("8 Vibrating alert");
                System.out.println("9 Screen saver");
            }
            case 6 -> {
                System.out.println("Settings:");
                System.out.println("1 Call settings");
                System.out.println("2 Phone settings");
                System.out.println("3 Security settings");
                System.out.println("4 Restore factory settings");
                System.out.print("Enter: ");
                int settings = input.nextInt();
                switch(settings) {
                    case 1 -> {
                        System.out.println("1 Automatic redial");
                        System.out.println("2 Speed dialling");
                        System.out.println("3 Call waiting options");
                        System.out.println("4 Own number sending");
                        System.out.println("5 Phone line in use");
                        System.out.println("6 Automatic answer");
                    }
                    case 2 -> {
                        System.out.println("1 Language");
                        System.out.println("2 Cell info display");
                        System.out.println("3 Welcome note");
                        System.out.println("4 Network selection");
                        System.out.println("5 Lights");
                        System.out.println("6 Confirm SIM service actions");
                    }
                    case 3 -> {
                        System.out.println("1 PIN code request");
                        System.out.println("2 Call barring service");
                        System.out.println("3 Fixed dialling");
                        System.out.println("4 Closed user group");
                        System.out.println("5 Phone security");
                        System.out.println("6 Change access codes");
                    }
                    case 4 -> System.out.println("Restore factory settings");
                }
            }
            case 7 -> System.out.println("Call divert");
            case 8 -> {
                System.out.println("Music:");
                System.out.println("1 Music player");
                System.out.println("2 Radio");
                System.out.println("3 Recorder");
                System.out.println("4 Track list");
                System.out.print("Enter: ");
                int music = input.nextInt();
                switch(music) {
                    case 1 -> System.out.println("Music player");
                    case 2 -> System.out.println("Radio");
                    case 3 -> System.out.println("Recorder");
                    case 4 -> System.out.println("Track list");
                }
            }
            case 9 -> System.out.println("Games:");
            case 10 -> System.out.println("Calculator:");
                
            case 11 -> System.out.println("Reminders");
            case 12 -> {
                System.out.println("Clock:");
                System.out.println("1 Alarm clock");
                System.out.println("2 Clock settings");
                System.out.println("3 Date setting");
                System.out.println("4 Stopwatch");
                System.out.println("5 Countdown timer");
                System.out.println("6 Auto update of date and time");
            }
            case 13 -> System.out.println("Profiles:");
            case 14 -> System.out.println("Services");
            case 15 -> System.out.println("SIM Services");
        }
    }
}

File: menu_list.py

print("1. Phone book")
print("2. Messages")
print("3. Chat")
print("4. Call register")
print("5. Tones")
print("6. Settings")
print("7. Call divert")
print("8. Music")
print("9. Games")
print("10. Calculator")
print("11. Reminders")
print("12. Clock")
print("13. Profiles")
print("14. Services")
print("15. SIM Services")

main = input("Enter main menu: ")

if main == "1":
    print("1 Search")
    print("2 Service Nos.")
    print("3 Add name")
    print("4 Erase")
    print("5 Edit")
    print("6 Copy")
    print("7 Assign tone")
    print("8 Send b'card")
    print("9 Options")
    print("10 Speed dials")
    print("11 Voice tags")
    phonebookoptions = input("Enter: ")
    
    if phonebookoptions == "1":
        print("Search")
    elif phonebookoptions == "2":
        print("Service Nos.")
    elif phonebookoptions == "3":
        print("Add name")
    elif phonebookoptions == "4":
        print("Erase")
    elif phonebookoptions == "5":
        print("Edit")
    elif phonebookoptions == "6":
        print("Copy")
    elif phonebookoptions == "7":
        print("Assign tone")
    elif phonebookoptions == "8":
        print("Send b'card")
    elif phonebookoptions == "9":
        print("1 Memory in use")
        print("2 Type of view")
        print("3 Memory status")
        option = input("Enter option: ")
        if option == "1":
            print("Memory in use")
        elif option == "2":
            print("Type of view")
        elif option == "3":
            print("Memory status")
    elif phonebookoptions == "10":
        print("Speed dials")
    elif phonebookoptions == "11":
        print("Voice tags")

elif main == "2":
    print("1 Write messages")
    print("2 Inbox")
    print("3 Outbox")
    print("4 Picture messages")
    print("5 Templates")
    print("6 Smileys")
    print("7 Message settings")
    print("8 Info service")
    print("9 Voice mailbox number")
    print("10 Service command editor")
    messagesettingsoptions = input("enter: ")

    if messagesettingsoptions == "1":
        print("Write messages")
    elif messagesettingsoptions == "2":
        print("Inbox")
    elif messagesettingsoptions == "3":
        print("Outbox")
    elif messagesettingsoptions == "4":
        print("Picture settings")
    elif messagesettingsoptions == "5":
        print("Templates")
    elif messagesettingsoptions == "6":
        print("Smileys")
    elif messagesettingsoptions == "7":
        print("Set 1")
        print("Common")
        option = input("Enter: ")
        if option == "1":
            print("Message centre number")
            print("Message sent as")
            print("Message validity")
        if option == "2":
            print("Delivery reports")
            print("Reply via same centre")
            print("Character support")
elif main == "3":
    print("Chat")

elif main == "4":
    print("1 Missed calls")
    print("2 Received calls")
    print("3 Dialled numbers")
    print("4 Erase recent call lists")
    print("5 Show call duration")
    print("6 Show call costs")
    print("7 Call cost settings")
    print("8 Prepaid credit")
    callregisteroption = input("Enter: ")

    if callregisteroption == "1":
        print("Missed calls")
    elif callregisteroption == "2":
        print("Received calls")
    elif callregisteroption == "3":
        print("Dialed numbers")
    elif callregisteroption =="4":
        print("Erase recent call lists")
    elif callregisteroption == "5":
        print("Last call duration")
        print("All calls' duration")
        print("Received calls' duration")
        print("Dialed calls' duration")
        print("Clear timers")
    elif callregisteroption == "6":
        print("Last call cost")
        print("All calls' cost")
        print("Clear counters")
    elif callregisteroption == "7":
        print("Call cost limit")
        print("Show costs in")
    elif callregisteroption == "8":
        print("Prepaid credit")
       
elif main == "5":
    print("1 Ringing tone")
    print("2 Ringing volume")
    print("3 Incoming call alert")
    print("4 Message alert tone")
    print("5 Keypad tones")
    print("6 Warning tones")
    print("7 Warning and game tones")
    print("8 Vibrating alert")
    print("9 Screen saver")

elif main == "6":
    print("1 Call settings")
    print("2 Phone settings")
    print("3 Security settings")
    print("4 Restore factory settings")
    settingsoption = input("Enter: ")
    
    if settingsoption == "1":
        print("Automated redial")
        print("Speed dialing")
        print("Call waiting options")
        print("Own number sending")
        print("Phone line in use")
        print("Automatic answer")
    elif settingsoption == "2":
        print("Language")
        print("Cell info display")
        print("Welcome note")
        print("Network selection")
        print("Confirm SIM service actions")
    elif settingsoption == "3":
        print("PIN code request")
        print("Call barring service")
        print("Fixed dialing")
        print("Closed user group")
        print("Security level")
        print("Change access codes")
    elif settingsoptions =="4":
        print("Restore factory settings")

elif main == "7":
    print("Call divert")

elif main == "8":
    print("1 Music player")
    print("2 Radio")
    print("3 Recorder")
    print("4 Track list")

elif main == "9":
    print("Games")

elif main == "10":
    print("Calculator")

elif main == "11":
    print("Reminders")

elif main == "12":
    print("Alarm clock")
    print("Clock settings")
    print("Date settings")
    print("Stopwatch")
    print("Countdown timer")
    print("Auto update of date and time")

elif main == "13":
    print("Profiles")

elif main == "14":
    print("Services")

elif main == "15":
    print("SIM Services")


